src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/document/InMemoryMultipartFile.java

```diff
@@ -79,7 +79,9 @@ public InputStream getInputStream() throws IOException {
 
     @Override
     public void transferTo(File dest) throws IOException, IllegalStateException {
-        new FileOutputStream(dest).write(payload);
+        try (FileOutputStream stream = new FileOutputStream(dest)) {
```

Comment:
Is this part of this PR ?

Reply:
Sonar failed the build because of that.

Reply:
it would be great to fix any sonar issues that are already there in the repo on a separate PR (and merge it first)

Reply:
Agreed #49